src/render.rs

```diff
@@ -124,17 +127,20 @@ gfx_defines! {
     }
 }
 
-fn get_shader(root: &str, name: &str, variant: &str) -> String {
+pub fn get_shader(root: &str, raw_path: &str, variant: &str) -> String {
```

Comment:
not strictly needed. `render` module is not public by itself, and this function is not re-exported, so it's not visible from the outside.